src/views/hidden-field.php

```diff
@@ -3,9 +3,26 @@
 		<?php
 		$hidden     = (array) get_post_meta( get_the_ID(), Tribe__Extension__Hide_Additional_Fields::$field_key, true );
 		$field_name = esc_attr( $customField['name'] );
+
+		/**
+		 * Allow filtering whether the checkboxes should be checked by default.
+		 *
```

Comment:
Let's add a `@since` here, to identify in which version the filter was added.

Reply:
Also, it'd be great it we cast the return of the filter as boolean

Reply:
Also, it'd be great it we cast the return of the filter as boolean.

Reply:
Done.